### PR TITLE
Use http to connect to docker in tests

### PR DESCRIPTION
**Pull request by podviaznikov**
This should fix the tests.
I reverted back connection to work over http in tests. Maybe it's not the best solution and we should make docker-mock to support https

### CODE REVIEW

File: configs/.env.test [hunk 1]
```diff
@@ -4,3 +4,4 @@ HOST_TAGS=some,comma,tags
 RABBITMQ_HOSTNAME=localhost
 RABBITMQ_PASSWORD=guest
 RABBITMQ_USERNAME=guest
+DOCKER_REMOTE_API_PROTOCOL="http"
```
Comment (bkendall) [inline configs/.env.test]: doesn't need to be in quotes

**Comment by bkendall** [comment]
booga booga! I see you typing!

**Comment by bkendall** [comment]
> we should make docker-mock to support https

oooo yeah! we should! :+1:

**Comment by bkendall** [comment]
this fixes master, so we should merge this. bad us, @anandkumarpatel

**Comment by bkendall** [comment]
LGTM for now - can do `docker-mock` https later :)

**Comment by bkendall** [comment]
:+1:

**Comment by cflynn07** [comment]
LGTM!

**Pull request merged**